package.json

```diff
@@ -1,27 +1,24 @@
 {
     "name": "caliper",
-    "description": "Performance benchmarking for Hyperledger blockchain technologies.  You must install [Lerna](https://lernajs.io) to build this multi-package repository.",
+    "description": "Performance benchmarking for Hyperledger blockchain technologies.",
     "version": "0.5.1-unstable",
     "repository": {
         "type": "git",
         "url": "https://github.com/hyperledger/caliper"
     },
     "scripts": {
-        "repoclean": "lerna clean",
-        "bootstrap": "lerna bootstrap",
-        "pretest": "npm run licchk",
+        "pretest": "npm run licchk --workspaces",
```

Comment:
Since we set `engine-strict=true`, trying to run `npm install` would throw an error if the version of npm is too low:
```txt
npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for caliper@0.5.1-unstable: wanted: {"node":">=14.19.0","npm":">=7.24.2"} (current: {"node":"14.19.3","npm":"6.14.17"})
npm ERR! notsup Not compatible with your version of node/npm: caliper@0.5.1-unstable
npm ERR! notsup Not compatible with your version of node/npm: caliper@0.5.1-unstable
npm ERR! notsup Required: {"node":">=14.19.0","npm":">=7.24.2"}
npm ERR! notsup Actual:   {"npm":"6.14.17","node":"14.19.3"}
```

I think we should leave it to the developer to install a version of npm that suits them the best as they may have some other projects that might break due to the version of npm we install through our script.

Reply:
ok, we'll try that and see how we get on. You'll need to submit a PR for the docs which is in the gh-pages branch and update the vNext and v0.5.0 docs.
I will put a comment on the caliper-contributors channel about what to expect

Reply:
The docs will need to explain the error they will receive and how to address it

Reply:
Sure, will update the docs in the next PR